libs/server/Storage/Functions/MainStore/ReadMethods.cs

```diff
@@ -19,11 +19,11 @@ public bool SingleReader(ref SpanByte key, ref RawStringInput input, ref SpanByt
                 return false;
 
             var cmd = input.header.cmd;
-            if ((byte)cmd >= CustomCommandManager.StartOffset)
+            if ((ushort)cmd >= CustomCommandManager.StartOffset)
```

Comment:
doesnt StartOffset neef to change? we were already overflowing at 200 , not sure how it was working, pls check.